common/params_pyx.pyx

```diff
@@ -0,0 +1,157 @@
+# distutils: langauge = c++
+import threading
+import os
+from libcpp cimport bool
+from libcpp.string cimport string
+from params_pxd cimport Params as c_Params
+
+cdef enum TxType:
+  PERSISTENT = 1
+  CLEAR_ON_MANAGER_START = 2
+  CLEAR_ON_PANDA_DISCONNECT = 3
+
+keys = {
```

Comment:
why are we only checking this in python?

Reply:
Do we really care? I could move it to C++ but not sure if it's worth the complexity.